src/components/cursor.js

```diff
@@ -250,6 +284,23 @@ module.exports.Component = registerComponent('cursor', {
       evt.preventDefault();
     }
 
+    if (this.data.rayOrigin === 'xrselect' && evt.type === 'selectstart') {
```

Comment:
Can this `if` be include in the [if above](https://github.com/aframevr/aframe/pull/5065/files#diff-0e82e2c917e7127027721c5ab28da273907fdf1ddebc653d68cda0a9f97b17a7R248)

```
  if (evt.type === 'selectstart') {
     this.activeXRInput = evt.inputSource;
      this.onMouseMove(evt);
      this.el.components.raycaster.checkIntersections();
      ....
  } 

```